piker/brokers/kraken.py

```diff
@@ -14,13 +14,13 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-"""
+'''
 Kraken backend.
 
-"""
+'''
 from contextlib import asynccontextmanager
 from dataclasses import asdict, field
-from typing import List, Dict, Any, Tuple, Optional
+from typing import Dict, List, Tuple, Any, Optional, AsyncIterator
```

Comment:
Ok so I wanted these type annots changed to 3.9+ style but i actually already did this in history in #247 😂 so maybe we can get the conflicts merged in via git then instead of you doing it manually.